js/web/lib/wasm/binding/ort-wasm.d.ts

```diff
@@ -17,7 +21,10 @@ export declare namespace JSEP {
   type CaptureEndFunction = () => void;
   type ReplayFunction = () => void;
 
-  export interface Module extends WebGpuModule {
+  type AssociateContext = (sessionId: number, context: MLContext) => void;
```

Comment:
please:
- use `XxxxFunction` to be consistent with existing type name convension for function type.
- use `MlContext` instead of `Context` so that we know this is WebNN ML context.

Reply:
No longer applicable